### PR TITLE
Feature: revoke token

### PR DESCRIPTION
**Pull request by gaebel**
This adds token revocation (#24)

### CODE REVIEW

File: auth/build.gradle.kts [hunk 1]
```diff
@@ -240,6 +240,16 @@ publishing {
...
+                developer {
+                    id.set("benjohnde")
+                    name.set("Ben John")
+                    email.set("john@liftric.com")
+                }
+                developer {
+                    id.set("ingwersaft")
+                    name.set("Marcel Kesselring")
+                    email.set("kesselring@liftric.com")
+                }
```
Comment (gaebel) [inline auth/build.gradle.kts:252]: Hope this is fine for you @Ingwersaft @benjohnde ^^

File: auth/build.gradle.kts [hunk 2]
```diff
@@ -268,7 +278,7 @@ npmPublishing {
                     "aws"
                 )
                 licence = "MIT"
```
Comment (gaebel) [inline auth/build.gradle.kts:280]:
```diff
- licence
+ license
``` 
Probably the reason why this is missing at npmjs.

File: auth/src/commonTest/kotlin/com/liftric/auth/AuthHandlerIntegrationTests.kt [hunk 3]
```diff
@@ -199,6 +200,31 @@ abstract class AbstractAuthHandlerIntegrationTests() {
...
+    @Test
+    fun `Sign in, revoke token, validate`() = runTest {
+        val (result, credentials) = createUser()
+
+        val revokeTokenResponse = authHandler.revokeToken(result.RefreshToken)
+        assertNull(revokeTokenResponse.exceptionOrNull())
+        assertEquals(HttpStatusCode.OK, revokeTokenResponse.statusCode)
+
+        // AWS is not revoking Tokens instantly so give it some time
+        delay(1000)
```
Comment (benjohnde) [inline auth/src/commonTest/kotlin/com/liftric/auth/AuthHandlerIntegrationTests.kt:213]: 😏

**Comment by benjohnde** [comment]
LGTM! Happy to have targeted sign out on board soon :)

**Review by benjohnde — APPROVED** [review]

**Review by benjohnde — APPROVED** [review]
LGTM!

**Pull request merged**